main_GANs.py

```diff
@@ -5,8 +5,21 @@
         : Nouamane Laanait (e-mail: laanaitn@ornl.gov)
         : Simona Perotto (e-mail: simona.perotto@polimi.it)
 
+Usage:
+  main_GANs.py (-h | --help)
+  main_GANs.py --version
+  main_GANs.py [--epochs=100] [--optimizer='CGD'] [--learning_rate=0.0005]
+  main_GANs.py [-e 100] [-o 'CGD'] [-l 0.0005]
```

Comment:
Why do you need this line? Shouldn't the line above it be sufficient? What happens if they are inconsistent?

Reply:
This is an example of using short argument options. Note, the `=` is present in full version and absent in short.